packages/core/test/index.test.ts

```diff
@@ -0,0 +1,302 @@
+import { assert, expect } from 'chai';
```

Comment:
Most of these tests are all similar to tests that are happening in the `api/` package, just without the proxy and SDK chaining interface.